app/_src/using-mesh/managing-ingress-traffic/gateway-api.md

```diff
@@ -9,20 +9,26 @@ for configuring {% if_version gte:2.6.x inline:true %}[built-in gateway](/docs/{
 
 ## Installation
 
+{% if_version lte:2.6.x %}
 {% warning %}
 {{ site.mesh_product_name }}'s [Kubernetes Gateway API](https://gateway-api.sigs.k8s.io/) implementation is beta.
-
+{% endwarning %}
+{% warning %}
 Gateway API [`Gateways`](https://gateway-api.sigs.k8s.io/api-types/gateway/) aren't supported in multi-zone. To use the builtin Gateway, you need to use the [`MeshGateway` resources](/docs/{{ page.version }}/using-mesh/managing-ingress-traffic/builtin-listeners).
 {% endwarning %}
 
 1. Install the Gateway API CRDs.
 
-   The Gateway API CRDs aren't available in Kubernetes by default yet. You must first [install the _experimental release_ that includes those CRDs as well as the admission webhook](https://gateway-api.sigs.k8s.io/guides/#install-experimental-channel).
+   Kubernetes doesn't include Gateway API functionality by default. To use it, you'll need to install the CRDs first. Instructions: [Installing Gateway API Standard Channel](https://gateway-api.sigs.k8s.io/guides/#install-standard-channel).
```

Comment:
nitpick: to not duplicate this in line 30 everything above this could be inside `{% if_version lte:2.6.x %}` and start another block with `{% if_version lte:2.6.x %}` after this line.

Reply:
Then we would end with a numbered list containing only one element which I want to avoid

Reply:
why? we don't have to have the numbered item in the if, right? anyway, just a nitpick